metricbeat/helper/prometheus/prometheus.go

```diff
@@ -165,6 +166,14 @@ func (p *prometheus) GetProcessedMetrics(mapping *MetricsMapping) ([]common.MapS
 
 				// value may be a mapstr (for histograms and summaries), do a deep update to avoid smashing existing fields
 				update := common.MapStr{}
```

Comment:
Move this line and the comment above after the added code where is the related code.

Reply:
done!